com.civicrm.hrjobroles/js/src/job-roles/controllers/add-new-role.html

```diff
@@ -250,7 +250,9 @@
                         <ui-select-choices
                           prevent-animations
                           class="ui-select-choices"
-                          repeat="funder.value.value as (key,funder) in jobroles.FundersData | filter: { value: { title: $select.search }}">
+                          repeat="funder.value.value as (key,funder) in jobroles.fundersData | filter: { value: { title: $select.search }}"
+                          refresh="jobroles.getOptionGroupTypeValues('hrjc_funder')"
+                          refresh-delay="500">
```

Comment:
Why has it been added?

Reply:
The previous implementation has provision for refresh which was omitted in my initial implementation. It is needful as it ensure user always have updated list of funders (in case a new funder was added after page load by another user). The funders list also get updated when the tool icon is clicked but that is a longer route.